script/tool/test/drive_examples_command_test.dart

```diff
@@ -165,21 +154,20 @@ void main() {
     test(
         'driving under folder "test_driver" when targets are under "integration_test"',
         () async {
-      final Directory pluginDirectory = createFakePlugin('plugin', packagesDir,
-          withExtraFiles: <List<String>>[
-            <String>['example', 'test_driver', 'integration_test.dart'],
-            <String>['example', 'integration_test', 'bar_test.dart'],
-            <String>['example', 'integration_test', 'foo_test.dart'],
-            <String>['example', 'integration_test', 'ignore_me.dart'],
-          ],
-          isAndroidPlugin: true,
-          isIosPlugin: true);
+      final Directory pluginDirectory =
+          createFakePlugin('plugin', packagesDir, extraFiles: <List<String>>[
+        <String>['example', 'test_driver', 'integration_test.dart'],
```

Comment:
nit: it would be a bit more concise if this was just a list of strings that are paths, then we replaced the '/' with platform specific characters.  No need to address it here, just calling it out.
```dart
<String>[
        'example/test_driver/integration_test.dart',
        'example/integration_test/bar_test.dart',
        ...
```

Reply:
I think that may actually already work by accident, but I'll look at formalizing it. I agree the current system is needlessly messy.